--- ../original_verify/SL2ACJ.h	2020-10-31 15:05:42.626167000 -0400
+++ SL2ACJ.h	2020-10-31 15:05:42.680164000 -0400
@@ -24,7 +24,7 @@
 const int notIdentity(const SL2ACJ&x);
 const int notFPower(const SL2ACJ&x);
 const SL2ACJ shortGenerator(const ACJ&z);
-const SL2ACJ closeGenerator(const ACJ&x,const ACJ&x);
+const SL2ACJ closeGenerator(const ACJ&x,const ACJ&z);
 #endif
 
 /*:66*/
--- ../original_verify/ACJ.h	2020-10-31 15:05:42.598166000 -0400
+++ ACJ.h	2020-10-31 15:05:42.664171000 -0400
@@ -11,7 +11,7 @@
 #line 21 "ACJ.w"
 
 struct ACJ{
-ACJ(const XComplex&ff,const XComplex&ff0= 0,
+ACJ(const XComplex&ff= 0,const XComplex&ff0= 0,
 const XComplex&ff1= 0,const XComplex&ff2= 0,double err= 0)
 :f(ff),f0(ff0),f1(ff1),f2(ff2),e(err),
 size((1+2*EPS)*(absUB(ff0)+(absUB(ff1)+absUB(ff2))))
